packages/utils/src/carequality/cq-cert-checker.ts

```diff
@@ -0,0 +1,21 @@
+import express from "express";
+import * as https from "https";
+import * as fs from "fs";
+
+/**
+ * Script to test the Certificate issued by Carequality
+ * Run with ts-node run-cert.ts
+ */
+
+const app = express();
+
+// Read the private key and the chain of certificates
+const privateKey = fs.readFileSync("<path_to_your_private_key>/decrypted_private_key.key", "utf8");
+const certificateChain = fs.readFileSync("<path_to_your_chain>/chained_no_root.pem", "utf8");
+
+const credentials = { key: privateKey, cert: certificateChain };
+
+// Create HTTPS server
+https.createServer(credentials, app).listen(3000, () => {
+  console.log("Express server listening on port 3000 with HTTPS");
+});
```

Comment:
Curious, don't we need to log something if we successfully get a request and its validated properly?

Reply:
will address this on subsequent PR did not have time to figure this out. Currently you can look into the bundle returned by the openSSL command and see if it was verified or not.